src/utils/logger.js

```diff
@@ -0,0 +1,34 @@
+const pino = require("pino");
```

Comment:
Minor thing - perhaps we could `import` Pino rather than `require` it, to maintain consistency across the app? Not a big deal though.
```suggestion
import pino from "pino";
```

Reply:
Ah good catch, that'll be me from copying from courses-api :sweat_smile:. I've updated this in 4275374 but I've also taken it a step further and updated the export style as well so it's consistent